packages/client/src/runtime/core/engines/library/LibraryEngine.test.ts

```diff
@@ -90,7 +91,12 @@ jest.mock('fs', () => {
   const original = jest.requireActual('fs')
   return {
     ...original,
-    readFileSync: jest.fn().mockReturnValue(''),
+    readFileSync: (fileName, ...rest) => {
+      if (fileName === '/mock/schema.prisma') {
+        return ''
+      }
+      return original.readFileSync(fileName, ...rest)
```

Comment:
Fallback to original is now needed, because non-treeshaked `internals` will import `prisma-ft-wasm`, which will try to read wasm file, which will if fail if `readFileSync` is mocked uncondinitonally.